brd-ios/breadwallet/src/Helpers/SimpleTextField.swift

```diff
@@ -123,15 +147,27 @@ class SimpleTextField: UIView, UITextFieldDelegate {
         textField.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 4).isActive = true
         textField.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
         textField.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
-        textField.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
         textField.heightAnchor.constraint(equalToConstant: 48).isActive = true
         
+        addSubview(errorLabel)
+        errorLabel.topAnchor.constraint(equalTo: textField.bottomAnchor, constant: 4).isActive = true
+        errorLabel.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
+        errorLabel.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
+        errorLabel.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
```

Comment:
can you please share a screenshot that shows the error label?

Reply:
<img width="278" alt="Screenshot 2022-04-12 at 11 56 07" src="https://user-images.githubusercontent.com/40755142/162935747-ec1d9e4d-41bc-400f-b9a7-4a22556682d1.png">